frontend/src/components/pages/transforms/Transforms.List.tsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file https://github.com/redpanda-data/redpanda/blob/dev/licenses/bsl.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+/* eslint-disable no-useless-escape */
+import Section from '../../misc/Section';
+import { makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { appGlobal } from '../../../state/appGlobal';
+import { transformsApi } from '../../../state/backendApi';
+import PageContent from '../../misc/PageContent';
+import { PageComponent, PageInitHelper } from '../Page';
+// import { Box, Button, DataTable, SearchField, Text } from '@redpanda-data/ui';
+import { uiSettings } from '../../../state/ui';
+import { DefaultSkeleton } from '../../../utils/tsxUtils';
+import { Box, Button, DataTable, Flex, SearchField, Stack, Text } from '@redpanda-data/ui';
+import { Link as ReactRouterLink } from 'react-router-dom'
+import { Link as ChakraLink, createStandaloneToast } from '@chakra-ui/react';
+import { PartitionTransformStatus_PartitionStatus, TransformMetadata } from '../../../protogen/redpanda/api/dataplane/v1alpha1/transform_pb';
+import { CheckIcon } from '@chakra-ui/icons';
+import { XIcon } from '@heroicons/react/solid';
+import { Link } from 'react-router-dom';
+import { encodeURIComponentPercents } from '../../../utils/utils';
+import { openDeleteModal } from './modals';
+import { TrashIcon } from '@heroicons/react/outline';
+const { ToastContainer, toast } = createStandaloneToast();
+
+export function renderPartitionStatus(status: PartitionTransformStatus_PartitionStatus) {
```

Comment:
Can we make these components instead of a function? Sometimes it's hard to refactor once we will need to employ hooks here.

Reply:
done